easybuild/easyconfigs/a/angsd/angsd-0.921-foss-2018a.eb

```diff
@@ -0,0 +1,27 @@
+easyblock = 'MakeCp'
+
+name = 'angsd'
+version = '0.921'
+
+homepage = 'http://www.popgen.dk/angsd'
+description = """Program for analysing NGS data."""
+
+toolchain = {'name': 'foss', 'version': '2018a'}
+
+source_urls = ['https://github.com/ANGSD/angsd/archive/']
+sources = ['%(version)s.tar.gz']
+checksums = ['8892d279ce1804f9e17fe2fc65a47e5498e78fc1c1cb84d2ca2527fd5c198772']
+
+dependencies = [('HTSlib', '1.8')]
```

Comment:
@RvDijk We should stick to `1.7` here, since we already use `SAMtools` 1.7 in `*/2018a` easyconfigs.

See also https://github.com/easybuilders/easybuild-easyconfigs/pull/6454/files#diff-2e089e0f4425c911b806866fdb57c0acR23

Tests will start failing if both this PR and #6454 get merged because of different `HTSlib` versions being used as dependency in easyconfigs using a `2018a` toolchain.